src/plugins/kconfig/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+include (LibAddMacros)
+
+add_plugin (kconfig
+	    CPP
+	    ADD_TEST
+	    CPP_TEST
+	    TEST_README
+	    SOURCES kconfig.hpp
+		    kconfig.cpp
+		    kconfig_delegate.hpp
+		    kconfig_delegate.cpp
+		    kconfig_parser.cpp
+	kconfig_parser_exception.cpp
```

Comment:
Did you run `scripts/dev/reformat-cmake`?

Reply:
It re-formats every single `CMakeLists.txt` file in the repo.It might be because I have version `0.6.0` of `cmake-format` installed compared to `0.5.4` which is used by that script.

Reply:
https://www.libelektra.org/tutorials/run-reformatting-with-docker might be a good solution for you. Otherwise you would need to downgrade to cmake 0.5.4.